dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNBusInterfaceIIDM.cpp

```diff
@@ -138,6 +138,7 @@ BusInterfaceIIDM::importStaticParameters() {
   staticParameters_.insert(std::make_pair("Theta", StaticParameter("Theta", StaticParameter::DOUBLE).setValue(getAngle0())));
   staticParameters_.insert(std::make_pair("Upu", StaticParameter("Upu", StaticParameter::DOUBLE).setValue(getV0() / getVNom())));
   staticParameters_.insert(std::make_pair("Theta_pu", StaticParameter("Theta_pu", StaticParameter::DOUBLE).setValue(getAngle0() * M_PI / 180)));
+  staticParameters_.insert(std::make_pair("Vmax", StaticParameter("Vmax", StaticParameter::DOUBLE).setValue(getVMax())));
```

Comment:
Could you also add Vmin ?
Also, have you tested it ? Could you add it in the unit test ?

Reply:
Didn't we mention to add the Pu versions of the Vmin and Vmax too?

Reply:
Yes indeed